### PR TITLE
iptables: add Spanish translation

### PR DESCRIPTION
**Pull request by ikks**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/linux/iptables.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# iptables
+
+> Configura tablas, cadenas y reglas del firewall IPv4 del kernel Linux.
+> Utilice 'ip6tables' para establecer reglas para el tráfico IPv6. Vea también: `iptables-save`, `iptables-restore`.
```
Comment (acuteenvy) [inline pages.es/linux/iptables.md:4]:
```suggestion
> Utilice `ip6tables` para establecer reglas para el tráfico IPv6. Vea también: `iptables-save`, `iptables-restore`.
```

Comment (ikks) [reply pages.es/linux/iptables.md]: Thanks :)

File: pages.es/linux/iptables.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# iptables
+
+> Configura tablas, cadenas y reglas del firewall IPv4 del kernel Linux.
+> Utilice `ip6tables` para establecer reglas para el tráfico IPv6. Vea también: `iptables-save`, `iptables-restore`.
```
Comment (kant) [inline pages.es/linux/iptables.md:4]:
```suggestion
> Utiliza `ip6tables` para establecer reglas para el tráfico IPv6. Vea también: `iptables-save`, `iptables-restore`.
```

File: pages.es/linux/iptables.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+`sudo iptables --append {{cadena}} --source {{ip}} --protocol {{tcp|udp|icmp|...}} --dport {{puerto}} --jump {{regla}}`
+
+- Agrega una regla NAT para traducir todo el tráfico desde la subred `192.168.0.0/24` a la IP pública del host:
+
+`sudo iptables --table {{nat}} --append {{POSTROUTING}} --source {{192.168.0.0/24}} --jump {{MASQUERADE}}`
+
+- Borra la regla de la cadena:
+
+`sudo iptables --delete {{cadena}} {{numero_de_línea_de_regla}}`
```
Comment (kant) [inline pages.es/linux/iptables.md:29]:
```suggestion
`sudo iptables --delete {{cadena}} {{número_de_línea_de_regla}}`
```

**Review by kant — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**